components/image-builder-bob/README.md

```diff
@@ -28,7 +41,7 @@ docker push localhost:5000/source:latest
 ```
 
 Build and run
-```
+```bash
```

Comment:
```suggestion
```console
```

Reply:
thanks! wasn't aware of `console`, and always unsure whether to use `bash` or `shell`